clang/docs/ClangFormatStyleOptions.rst

```diff
@@ -4226,6 +4226,19 @@ the configuration (without a prefix: ``Auto``).
   ``ClassImpl.hpp`` would not have the main include file put on top
   before any other include.
 
+.. _IncludeSortKey:
```

Comment:
Did you write this on your own? You need to do that in `IncludeStyle.h` and run `clang/docs/tools/dump_format_style.py`.